sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/runsettings/ai_super_computer_configuration.py

```diff
@@ -2,13 +2,13 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
-from typing import Dict, List, Optional
+from typing import Any, Dict, List, Optional
 
 from ....entities._job.job_resource_configuration import BaseProperty
 
 
 class PascalCaseProperty(BaseProperty):
-    _KEY_MAPPING = {}
+    _KEY_MAPPING: Dict[str, Any] = {}
```

Comment:
It might be a good idea not to even have a default for `_KEY_MAPPING` here in the base class; otherwise it's very easy to accidentally mutate the mapping that'd then be shared by each class inheriting from `PascalCaseProperty` that doesn't re-declare its own `KEY_MAPPING`.

Also, I believe the type is `Dict[str, str]`?

```suggestion
    _KEY_MAPPING: Dict[str, str]  # set in subclasses
```

Reply:
On the first point, fair suggestion, but I'll let the code owners make that change if they like. I didn't write the original code, my team is only responsible for getting the code mypy-compliant for now, so that's out of scope of this PR.

On the second, I used Dict[str, Any] because some of the subclasses of BaseProperty use Enums as aliases for those strings and sometimes mypy won't recognize those as strings, but instead as the Enum class name like the example here:

![image](https://github.com/Azure/azure-sdk-for-python/assets/16376603/3733e801-5e32-471b-9e3d-4cca8e0cf9a6)